knodle/trainer/config.py

```diff
@@ -30,6 +31,7 @@ def __init__(
             caching_suffix: str = "",
             saved_models_dir: str = None
     ):
+        self.seed = seed
```

Comment:
Do you need this to seed CleanLab? I also wanted to add this line for Snorkel, so nothing against it, just curious

Reply:
yes! Cleanlab takes seed as an input parameter